addons/binding/org.openhab.binding.vitotronic/ESH-INF/binding/binding.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <binding:binding id="vitotronic"
-        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
-        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+                 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"

```

Comment:
yes, I replaced all tabs by 4 spaces. 
I did this because the travis build alwasy failed and the error message was that there were tabs in the files.
After replacing the tabs by spaces the build went well